utils/routine.go

```diff
@@ -0,0 +1,38 @@
+package utils
+
+import (
+	"sync"
+
+	"github.com/ercole-io/ercole-agent/config"
+)
+
+// RunRoutine will run function in a separeted goroutine and wait or not due to config.
+func RunRoutine(configuration config.Configuration, function func()) {
+	done := make(chan bool, 1)
+
+	go func(done chan<- bool) {
+		function()
+		done <- true
+	}(done)
+
+	if !configuration.ParallelizeRequests {
+		<-done
+	}
```

Comment:
Why do you the done channel and don't simply call function() if !configuration.ParallelizeRequest?

Reply:
Fixed! Thanks!